cloud-haven-react-app/src/components/ProfileDropdown/ProfileDropdown.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Dropdown from 'react-bootstrap/Dropdown'
+import './ProfileDropdown.css';
```

Comment:
```suggestion
```
Looks like the file `ProfileDropdown` is empty, so let's remove this. I'll click "delete" on the file